### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by eljo**
Fought with rollbacks where convox logs didn't give a reason for hours.

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _help/troubleshooting.md [hunk 1]
```diff
@@ -19,3 +19,7 @@ Run `convox apps info` to find the load balancer endpoints for your application.
 Run `convox ps` to determine if your application is booting successfully.
 
 Run `convox logs` to inspect your application logs and cluster events for problems placing your container, starting your app, or registering with the load balancer.
+
+## My app deployed but immediately exits and rolls back
+
+If your application rolls back for apparently no reason (i.e. no error is given on a `convox deploy`) check your CloudFormation events for details -- it might be a simple typo in your convox.cron expression.
```
Comment (mwarkentin) [inline _help/troubleshooting.md]: The cloudformation logs should be available in `convox logs` as of the latest release as well.

Comment (eljo) [inline _help/troubleshooting.md]: All I got was a reason="" for a mistyped convox.cron rollback, tested about 2hrs ago. Will test again.

Comment (eljo) [inline _help/troubleshooting.md]: Confirming.
The first log message after an expected one ("2017-03-07T21:26:21Z web:RRNWUKMFDPK/8653f0147fdf Use Ctrl-C to stop") is:

2017-03-07T21:27:27Z CloudformationEvent resource="CronFunction" status="DELETE_IN_PROGRESS" reason=""

(a proper reason wouldn't hurt of course)

**Comment by soulshake** [comment]
@eljo Any way you could open a support ticket and provide the exact cron you were running so we can try to reproduce this and improve the error reporting?

**Pull request closed without merging**